common/src/main/java/org/apache/uniffle/common/metrics/GRPCMetrics.java

```diff
@@ -119,4 +138,12 @@ public Gauge getGaugeGrpcOpen() {
   public Counter getCounterGrpcTotal() {
     return counterGrpcTotal;
   }
+
+  public Map<String, Summary> getSendTimeSummaryMap() {
```

Comment:
What's the meaning of `sendTime`?

Reply:
Could we have a better name?

Reply:
Its meaning is the time interval from the client sending to the ShuffleServerGrpcService receiving the request.

Reply:
> Could we have a better name? Transport time?

Sounds great!